tools/python_api/test/conftest.py

```diff
@@ -28,7 +28,7 @@ def init_tiny_snb(tmp_path):
                  "STRING[], stock STRUCT(price INT64[], volume INT64)), PRIMARY KEY (ID));")
     conn.execute('COPY organisation FROM "../../../dataset/tinysnb/vOrganisation.csv"')
     conn.execute('CREATE NODE TABLE movies (name STRING, length INT32, note STRING, description STRUCT(rating DOUBLE, '
-                 'views INT64, release TIMESTAMP, film DATE), PRIMARY KEY (name))')
+                 'views INT64, release TIMESTAMP, film DATE), content BYTEA, PRIMARY KEY (name))')
```

Comment:
You can leave this to others. But we should just let python test to read from `schema.cypher and copy.cypher` so that we don't need to change this. I guess there is an issue about refactor python test and you can add this to that issue